staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization_test.go

```diff
@@ -127,3 +133,60 @@ func TestGetAuthorizerAttributes(t *testing.T) {
 		}
 	}
 }
+
+// fakeAuthorizer allows all reqeusts from admin, and denies all requests from tom.
+// an error is returned for other users.
+type fakeAuthorizer struct{}
+
+func (fakeAuthorizer) Authorize(a authorizer.Attributes) (authorizer.Decision, string, error) {
```

Comment:
nit: I would simplify this to:
```go
type fakeAuthorizer struct{
  decision authorizer.Decision
  reason string
  err error
}

func (f *fakeAuthorizer) Authorize(a authorizer.Attributes) (authorizer.Decision, string, error) {
  return f.decision, f.reason, f.err
}

...
```

And then make the test set the expected values.

(The less logic in the test construct the better)

Reply:
Done.